Sources/ApolloSQLite/SQLiteNormalizedCache.swift

```diff
@@ -127,6 +132,10 @@ extension SQLiteNormalizedCache: NormalizedCache {
     return try mergeRecords(records: records)
   }
   
+  public func removeRecord(for key: CacheKey) throws {
+    try removeSQLiteRecord(for: key)
```

Comment:
Why the separate `removeSQLiteRecord()`? Wouldn't `removeRecord()` on the SQLite cache already imply that?

Reply:
to match the other things in this file that have a wrapper method